2021/43xxx/CVE-2021-43309.json

```diff
@@ -1,18 +1,84 @@
 {
```

Comment:
'' does not match '^([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\\.([a-zA-Z]{2,5})$'

Reply:
Fixed in new version